neuraxle/metaopt/auto_ml.py

```diff
@@ -863,15 +879,15 @@ def _load_virgin_model(self, hyperparams: HyperparameterSamples) -> BaseStep:
 
 def _get_trial_split_description(
         repo_trial: Trial,
-        repo_trial_split: TrialSplit,
+        repo_trial_split_number: int,
```

Comment:
Looking at this twice quickly, something looks weird with this method. It feels like it should be a member of the repo trial split class. Anyway, good for now.